x-pack/plugins/maps/public/angular/map_controller.js

```diff
@@ -205,6 +206,13 @@ app.controller('GisMapController', ($scope, $route, config, kbnUrl, localStorage
     { text: $scope.map.title }
   ]);
 
+  chrome.helpExtension.set(domElement => {
```

Comment:
Maybe this extension point also needs to be set in https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/index.js so that the help context is also displayed on the dashboard listing page

<img width="2043" alt="screen shot 2019-03-06 at 10 50 20 am" src="https://user-images.githubusercontent.com/373691/53902112-b8ba3e80-3ffd-11e9-8932-f91e0e962ea5.png">